root/static/scripts/release/components/ReleaseRelationshipEditor.js

```diff
@@ -398,11 +407,6 @@ async function submitRelationshipEdits(
         editData.gid = entity.gid;
       }
       return editData;
-    } else if (entity.entityType === 'work') {
-      invariant(
-        entity._fromBatchCreateWorksDialog !== true &&
-        isDatabaseRowId(entity.id),
-      );
```

Comment:
This invariant checked that batch-created works were submitted (i.e. created) before any relationships for them were, but it has to be removed because that can't be true before any edits are submitted.

Reply:
Do we still check that in any way?

Reply:
Well, it's checked by /ws/js/edit, so the request would fail.  But that's probably better since it would show an error message instead of crashing the whole page.

Reply:
Can the user do anything if that happens? I assume the whole thing is sorted to happen in the right order elsewhere anyway and this just made sure bugs were detected? In which way detecting them via ws/js/edit seems fine too, at least if the error message returned is easy to report.

Reply:
Yeah, the submission is done in the right order, so I was mostly using it for debugging.

It should stop submitting edits as soon as a /ws/js error occurs, then show whatever error message it got at the bottom of the page near the edit note, and allow you to re-submit from where it left off (in case removing the offending edits helps).